app/.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": false
```

Comment:
- Looks like `useTabs` is [`false` by default](https://prettier.io/docs/en/options.html#tabs) and `tabWidth` is set to [2 by default](https://prettier.io/docs/en/options.html#tab-width). Do we need to explicitly state them here?
- Consider [`"trailingComma": "all"`](https://prettier.io/docs/en/options.html#trailing-commas) to match CA UI and Mass PFML